### PR TITLE
Further align shebangs with code that has executable entry points

### PR DESCRIPTION
**Pull request by leondz**
continuation of #612

### CODE REVIEW

**Review by jmartin-tech — APPROVED** [review]
LGTM

File: garak/analyze/aggregate_reports.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
+
```
Comment (jmartin-tech) [inline garak/analyze/aggregate_reports.py:2]: These additions in utility classes may not provide the utility intended. Happy to leave them for now.

Would you ever expect a user to execute:
```
./aggregate_reports.py [args]
```

**Review by jmartin-tech — COMMENTED** [review]
Question, looking at the `added` entires, I suspect these are not really usable.

File: garak/__main__.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
+
```
Comment (jmartin-tech) [inline garak/__main__.py:2]: These additions in utility classes may not provide the utility intended. Happy to leave them for now.

Would you ever expect a user to execute:
```
./__main__.py -m garak --model_type huggingface --model_name gpt2 --probes dan.Dan_11_0
```

Comment (leondz) [reply garak/__main__.py:2]: No. OTOH cli.py isn't intended to invoked either. Let's remove them both.

File: garak/analyze/aggregate_reports.py (hunk 1 above)
Comment (leondz) [reply garak/analyze/aggregate_reports.py:2]: yes, definitely!

**Pull request merged**